js/src/dapps/registry/Events/events.js

```diff
@@ -149,6 +221,13 @@ export default class Events extends Component {
             onToggle={ this.onDataChangedToggle }
             style={ inlineButton }
           />
+          <Toggle
```

Comment:
The whole toggle concept here just makes things more complicated that it needs to be. First time I do each time I use the app is to click all the toggles so I can actually see what is happening when I perform actions.